azurelinuxagent/agent.py

```diff
@@ -208,28 +209,30 @@ def collect_logs(self, is_full_mode):
         else:
             logger.info("Running log collector mode normal")
 
+        LogCollector.initialize_telemetry()
+
         # Check the cgroups unit
         log_collector_monitor = None
         tracked_controllers = []
         if CollectLogsHandler.is_enabled_monitor_cgroups_check():
             try:
                 cgroup_api = get_cgroup_api()
             except InvalidCgroupMountpointException as e:
-                log_cgroup_warning("The agent does not support cgroups if the default systemd mountpoint is not being used: {0}".format(ustr(e)), send_event=True)
+                log_cgroup_warning("The agent does not support cgroups if the default systemd mountpoint is not being used: {0}".format(ustr(e)), op=WALAEventOperation.LogCollection, send_event=True)
```

Comment:
Updating these events to be marked as LogCollection operations

Reply:
I understand the motivation, but odd to send logcollection events with a function named as "cgroup". Plus the local log will still have the cgroups mark ("[CGW"]).

Maybe just create a similar log_log_collection_warn function? or if you don't need the special mark on the local log just use event.warn?